src/blockstore.js

```diff
@@ -49,6 +50,21 @@ function maybeWithSharding (filestore, options, callback) {
 
 function createBaseStore (store) {
   return {
+    /**
+     * Query the store.
+     *
+     * @param {object} query
+     * @param {function(Error, Array)} callback
+     * @return {void}
+     */
+    query (query, callback) {
+      pull(
+        store.query(query),
+        pull.collect((err, list) => {
```

Comment:
I think you can pass callback directly to `pull.collect` here.

Reply:
Nice catch! Thanks 😄